crypto/rust/src/lib.rs

```diff
@@ -626,6 +626,8 @@ impl_hash_type!(KESVkey, 32);
 // same for this signature (but lots of traits aren't implemented for [u8; 448] so we can't)
 //impl_hash_type!(KESSignature, 448);
 impl_hash_type!(NonceHash, 32);
+// not sure if this is an existing hash or not. made it a new one but maybe it's datum hash or something?
```

Comment:
🤔

Reply:
As far as I can tell it's its own new thing. The formal spec refers to it as a DocHash (hash of the contents at the url in the anchor pair). I'll rename it based on what I see in the formal spec for now, as that makes more sense than the name in the CDDL spec.